kibana-reports/public/components/main/main_utils.tsx

```diff
@@ -63,7 +63,7 @@ export const addReportsTableContent = (data) => {
       //TODO: wrong name
       timeCreated: report.time_created,
       state: report.state,
-      url: report.query_url,
+      url: `${location.host}${report.query_url}`,
```

Comment:
What does `location.host` add?

Reply:
the `query_url` is now relative, need to add the host as well for the link rendered in report table -> report source field to be accessible.